Makefile

```diff
@@ -6,3 +12,11 @@ package:
 deploy:
 	$(MAKE) -C images deploy
 
+.PHONY: import
+import: package
+	-gravity app delete --ops-url=$(OPS_URL) $(REPOSITORY)/$(NAME):$(VERSION) \
+		--force --insecure
+	gravity app import --vendor --glob=**/*.yaml --registry-url=apiserver:5000 \
+		--ops-url=$(OPS_URL) --repository=$(REPOSITORY) --name=$(NAME) \
+		--version=$(VERSION) --insecure .
```

Comment:
I would also add `--set-image=log-collector:$(VERSION)` and `--set-image=log-forward:$(VERSION)` to set proper image names during build.

This also means that during container builds you'd need to tag resulting image both as `log-forward:$(VERSION)` - to vendor it and `log-forward:latest` to use in development mode directly.